src/root/file.d

```diff
@@ -73,7 +73,7 @@ struct File
             size_t size;
             stat_t buf;
             ssize_t numread;
-            char* name = this.name.toChars();
+            const(char)* name = this.name.toChars();
```

Comment:
That will mean we're going to have to go back and change them all again when `toChars` starts returning `string`